source/administration/command-line-tools.rst

```diff
@@ -894,8 +894,32 @@ mattermost export schedule
           --exportFrom string     Unix timestamp (seconds since epoch, UTC) to export data from.
           --timeoutSeconds string Set how long the export should run for before timing out.
 
+mattermost extract-documents-content 
+-------------------------------------
+
+  Description
+    Extracts document content and stores it in the database for file searches available in Mattermost Cloud, and in Mattermost Server from v5.35. This database migration is strongly recommended so that `users can search for file attachments <https://docs.mattermost.com/help/getting-started/searching.html#searching-for-files>`__ shared in Mattermost prior to v5.35. Otherwise, search results for files shared in the past may be incomplete. The database migration command extracts and indexes the content of past file attachments.
```

Comment:
I don't know if I would call it "database migration" is not exactly that. Also, if makes file searches by content available, but if you don't execute it you still can search by filename (when you are using database). If you are using elasticsearch/bleve, you also needs to execute a reindex of elasticsearch/bleve.

Reply:
Thanks, @jespino. I've updated descriptions of the CLI command to be an extraction command, rather than a DB migration command. I've also added notes about users being able to search by file name only if this CLI command is not executed.

Reply:
@cwarnermm that's a good point we should add here (that if you're using Elasticsearch or Bleve you also have to re-index)

Also perhaps we should also say "This database migration is strongly recommended" > "Running this content extraction command is strongly..."